Source/Checkers.cs

```diff
@@ -84,7 +85,8 @@ public static bool IsUnityCompatible()
         }
 
         // Version of the compatibility checker itself.
-        private static int _version = 5;
+
```

Comment:
Just to separate the var from the function for easier readability. I can supress it if you want.